releasenotes-dca/notes/inject_language_libraries_in_admission_controller-63eae7f820cc0f45.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG-DCA.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Enable `APM` library injection in `cluster-agent` admission controller based on automatic language detection annotations.
```

Comment:
could you add in the description that it is a Beta feature, and we might introduce some breaking change in the next agent version.